### PR TITLE
Run sidecar injector as root.

### PR DESCRIPTION
**Pull request by Yannig**
Run sidecar injector as root.

Fix https://github.com/istio/istio/issues/10358

### CODE REVIEW

**Comment by Yannig** [comment]
I signed it!

File: install/kubernetes/helm/istio/templates/sidecar-injector-configmap.yaml [hunk 1]
```diff
@@ -49,6 +49,7 @@ data:
             cpu: 10m
             memory: 10Mi
         securityContext:
+          runAsUser: {{ .Values.global.proxy.runAsUser }}
```
Comment (rshriram) [inline install/kubernetes/helm/istio/templates/sidecar-injector-configmap.yaml]: you might as well set this as root directly.

Comment (Yannig) [reply install/kubernetes/helm/istio/templates/sidecar-injector-configmap.yaml]: Done.

**Comment by sdake** [comment]
The issue for this problem was closed.  Did you intend to close the PR ras well?

Cheers
-steve

**Comment by Yannig** [comment]
@sdake from my point of view, this problem is still present.

Actually, to deploy my application with Istio, I need to patch the istio chart.

Regards,

Yannig

**Pull request closed without merging**